katas/content/single_qubit_gates/prepare_arbitrary_state/solution.md

```diff
@@ -0,0 +1,8 @@
+### Solution
```

Comment:
Similarly to the other comment, markdowns that represent explained solutions are known to be so and it is better they don't have a title.